msys2-runtime-3.4/update-patches.sh

```diff
@@ -21,8 +21,10 @@ base_tag=refs/tags/cygwin-"$(sed -ne 's/^pkgver=//p' <PKGBUILD)"
 msys2_branch=refs/heads/msys2-${base_tag#refs/tags/cygwin-}
 url=https://github.com/msys2/msys2-runtime
 
-test -d msys2-runtime ||
-git clone --bare $url msys2-runtime ||
+test -d msys2-runtime || {
```

Comment:
I wonder if it wouldn't be easier to just name the clone differently to avoid any conflicts. But maybe I'm missing something.

Reply:
I do want to reuse the clone, as it is not exactly a small repository.

Reply:
ok, fair enough